p3dx:
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50
        
    left_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: left_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0}

    right_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: right_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0}
